tests/tensor_base/test_augmented_updates.py

```diff
@@ -0,0 +1,111 @@
+from numpy.testing import assert_allclose
+import mygrad as mg
+import numpy as np
+
+
+# TODO: These tests need to be modified to check that tensor-identity
+# is preserved by the augmented update once #280 is resolved
+# https://github.com/rsokl/MyGrad/pull/280/
+def test_augmented_multiply():
```

Comment:
Missing imatmul?

Reply:
Thanks for reminding me about this. I decided to block users from using `__imatmul__` until numpy supports it